docs/operate-and-deploy/installation/server-config/config-reference.md

```diff
@@ -320,10 +321,10 @@ For more information, refer to the
 [CREATE STREAM AS SELECT](../../../developer-guide/ksqldb-reference/create-stream-as-select.md)
 statements.
 
-!!! note
-	 The `DELIMITED` format is not affected by the
-    `ksql.persistence.ensure.value.is.struct` setting, because it has no
-    concept of an outer record or structure.
+ !!! note
+   Not all formats support wrapping and unwrapping. If you use a format that does not support
+   the default value you set, the format will ignore the setting. For information on which formats
```

Comment:
It doesn't look like the docs specify what the default value for each format is.

Also, where is the default added in the code right now? Is it hard-coded to be wrapped (if SerdeOptions is empty) somewhere in the planner?

Reply:
> It doesn't look like the docs specify what the default value for each format is.

Yep, that's on my todo list - next PR!

> Also, where is the default added in the code right now? Is it hard-coded to be wrapped (if SerdeOptions is empty) somewhere in the planner?

If the user doesn't supply the option then nothing is stored in the plan.  Currently, this defaults to wrapped, (code in GenericRowSerde and GenericKeySerde), but the next PR will change this, so that the format sets the default.